var/spack/repos/builtin/packages/python/package.py

```diff
@@ -303,6 +305,60 @@ def determine_variants(cls, exes, version_str):
             except ProcessError:
                 variants += '~tix'
 
+        # Get config args for shared, debug, and optimizations variant checks
+        sysconf_imp = 'import sysconfig'
+        if version < Version('2.7'):
+            sysconf_imp = 'import distutils.sysconfig as sysconfig'
+        conf_args = python(
+            '-c',
+            '%s; print(sysconfig.get_config_var("CONFIG_ARGS"))' % sysconf_imp,
+            output=str, error=str)
```

Comment:
See `self.get_config_var`. It's hard-coded to use `self.command`, but maybe we could modify it so there is less overlap. At the very least, this print syntax won't work for older versions of Python. That's why we use `self.print_string` elsewhere.

Reply:
I thought as long as you were printing a single value, `print(foo)` and `print foo` were the same at least as far back as 2.6. I only have 2.7 around to confirm on though. Do you have easy access to 2.6 or should I build it to test?

Reply:
I guess they are the same, as `(foo)` will be treated as `foo`.